complete/src/main/java/hello/CoffeeLoader.java

```diff
@@ -0,0 +1,31 @@
+package hello;
+
+import org.springframework.data.redis.connection.ReactiveRedisConnectionFactory;
+import org.springframework.data.redis.core.ReactiveRedisOperations;
+import org.springframework.stereotype.Component;
+import reactor.core.publisher.Flux;
+
+import javax.annotation.PostConstruct;
+import java.util.UUID;
+
+@Component
+public class CoffeeLoader {
+    private final ReactiveRedisConnectionFactory factory;
+    private final ReactiveRedisOperations<String, Coffee> coffeeOps;
+
+    public CoffeeLoader(ReactiveRedisConnectionFactory factory, ReactiveRedisOperations<String, Coffee> coffeeOps) {
+        this.factory = factory;
+        this.coffeeOps = coffeeOps;
+    }
+
+    @PostConstruct
+    public void loadData() {
+        factory.getReactiveConnection().serverCommands().flushAll().thenMany(
```

Comment:
That's fine here. The code isn't blocking and Spring Data Redis does not use `ApplicationEvents`.